scipy/optimize/_differentialevolution.py

```diff
@@ -634,8 +732,7 @@ def solve(self):
             result = minimize(self.func,
                               np.copy(DE_result.x),
                               method='L-BFGS-B',
-                              bounds=self.limits.T,
-                              args=self.args)
```

Comment:
I don't see so quickly why `args` is no longer passed in?

Reply:
L484, `self.func = _FunctionWrapper(func, args)`. The `_FunctionWrapper` supplies any extra arguments. Wrapping is necessary for `map-likes` to work.